Examples/Python/tests/test_examples.py

```diff
@@ -1360,14 +1361,20 @@ def test_exatrkx(tmp_path, trk_geo, field, assert_root_hash, backend, hardware):
     root_file = "performance_track_finding.root"
     assert not (tmp_path / root_file).exists()
 
-    if backend == "onnx":
-        url = "https://acts.web.cern.ch/ci/exatrkx/onnx_models_v01.tar"
-    else:
-        url = "https://acts.web.cern.ch/ci/exatrkx/torchscript_models_v01.tar"
+    # Extract both models, since we currently don't have a working implementation
+    # of metric learning with ONNX and we need to use torch here
+    onnx_url = "https://acts.web.cern.ch/ci/exatrkx/onnx_models_v01.tar"
+    torch_url = "https://acts.web.cern.ch/ci/exatrkx/torchscript_models_v01.tar"
+
+    for url in [onnx_url, torch_url]:
+        tarfile_name = tmp_path / "models.tar"
+        urllib.request.urlretrieve(url, tarfile_name)
+        tarfile.open(tarfile_name).extractall(tmp_path)
+
+    shutil.copyfile(
+        tmp_path / "torchscript_models/embed.pt", tmp_path / "onnx_models/embed.pt"
+    )
```

Comment:
Maybe not the right place to do this, but we might want to externalize the download here into a separate script that runs before the tests do. In any case, we should add a sha256 hash sum check for the downloaded files.

Reply:
Yeah that makes sense. Can we do this in a separate PR? I can also open an issue on it to keep track on this...

Reply:
Sure.